proto/internalapi/central/collector_runtime_config.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package central;
+
+import "google/protobuf/timestamp.proto";
+import "storage/resource_collection.proto";
+import "storage/user.proto";
```

Comment:
This is a new message and as such should not include anything from `storage` per the [API Guidelines](https://github.com/stackrox/stackrox/blob/master/.github/api_guidelines.md#decoupling)

So you would need to replicate those objects here or in a common proto that is not in `storage`.  Yes it is duplication, but it is best practice to separate the APIs from the storage so they can change independently.

Reply:
I have created a `common` directory and duplicated everything I need from storage there.